galaxy_ng/app/auth/auth.py

```diff
@@ -58,6 +64,16 @@ def authenticate(self, request):
 
         return user, {'rh_identity': header}
 
+    def _ensure_group(self, account_scope, account):
+        """Create a auto group for the account and create a synclist distribution"""
+
+        with transaction.atomic():
+            group, created = Group.objects.get_or_create_identity(account_scope, account)
+            upstream_repo = self._get_upstream_repo(group, DEFAULT_UPSTREAM_REPO_NAME)
```

Comment:
This needs to create a synclist for the user as well as a repo and distribution.

Reply:
For the user?  I'm not sure I understand.

Will that be a synclist per user? in addition to per account_number / group?
First thought is that might double (or more) the number of synclists and the numbers of synclists tasks then need to get run.

But I think I'm missing details.  Would that also imply a per user synclist distribution? ie, a distro with a base_path like '{account_number}-{user}-synclist'   "99999-billybob-synclist"? 

What will per user synclists be used for?

Reply:
Updated to create a default synclist as side effect of auth (wherein it was previously only creating the synclist repo and distribution)